README.md

```diff
@@ -1,9 +1,31 @@
-Dune - A Composable Build System
-================================
 
-[![Main workflow][workflow-badge]][workflow]
-[![Release][release-img]][release]
-[![Coverage Status][coverall-badge]][coverall]
+
+<p align="center">
```

Comment:
HTML is ignored when doing `gh repo view`. For example:
- https://github.com/janestreet/magic-trace
- https://github.com/NixOS/nixpkgs
Both have html, but `gh repo view` ignored this.

Reply:
And from what I've seen on GitHub, centering is pretty typical when you have a header image.

Reply:
I just noticed that you wanted to have the badges available from `gh repo view`, I'll see what I can do.

Reply:
`gh repo view` is just an example - READMEs are consumed by many tools. They should make sense as text. What does the logo look like with a markdown image?

Reply:
OK I've pushed the markdown version now. Perhaps it is not too bad even on squashed screens.

Reply:
Thanks. Yes, I agree it's too large. One suggestion: how about embedding a copy of the logo resized to 252x63? This will open the possibility of including it in the sphinx docs too (it renders at that size).